include/deal.II/lac/trilinos_solver.h

```diff
@@ -364,24 +358,8 @@ namespace TrilinosWrappers
   class SolverCG : public SolverBase
   {
   public:
-    /**
-     * Standardized data struct to pipe additional data to the solver.
-     */
 
-    struct AdditionalData
-    {
-      /**
-       * Set the additional data field to the desired output format.
-       */
-      explicit
-      AdditionalData (const bool output_solver_details = false);
-
-      /**
-       * Enables/disables the output of solver details (residual in each
-       * iterations etc.).
-       */
-      bool output_solver_details;
-    };
+    using SolverBase::AdditionalData;
```

Comment:
You need to add this line to all of the other solver classes as well. It would also be good to keep the documentation.